scripts/semantic-commits/get-binary-release-data.js

```diff
@@ -133,7 +133,12 @@ const getReleaseData = async (latestReleaseInfo) => {
       return
     }
 
-    const ref = references.find((r) => !r.raw.includes('revert #'))
+    // Make sure the PR merged in is not a revert and
+    // the reference in question is the squashed commit merged into develop.
+    // This will follow the format 'TYPE: SUBJECT (#PR_REQUEST_NUMBER).
+    // This is to eliminate multiple references in a single commit that might point to an issue and not a PR.
+    // NOTE: this might cause issues with rebase merge commits into develop that have multiple references, which should rarely happen
+    const ref = references.find((r) => !r.raw.includes('revert #') && (references.length > 1 ? r.raw.match(/\(#\d*/) : true))
```

Comment:
What if the commit message references the issue between parentheses, like `some change (#12345)` and there's no associated PR? We'd end up in the same situation. I wonder if we should update the "Semantic PR" check to disallow issue references within the PR title.

Reply:
I think thats a good idea. Once we get the job back up and running I think it's something we should explore.

Reply:
OK I took another approach to this like we talked about where we will try to look up valid references for a given commit until we find a PR. This approach is a lot safer than the previous one taken